src/linker/Linker/TypeMapInfo.cs

```diff
@@ -299,9 +300,14 @@ IEnumerable<InterfaceImplementation> GetDefaultInterfaceImplementations (TypeDef
 
 		MethodDefinition TryMatchMethod (TypeReference type, MethodReference method)
 		{
+			var isStaticInterfaceMethod = (method as MethodDefinition)?.IsStatic;
 			foreach (var candidate in type.GetMethods (context)) {
 				var md = context.TryResolve (candidate);
-				if (md?.IsVirtual != true)
+				//Static interface methods - they are not virtual
+				bool isMethodStaticInterface = false;
```

Comment:
The naming is confusing - what's the difference between `isStaticInterfaceMethod` and `isMethodStaticInterface`? They both read the same to me.